storage.go

```diff
@@ -760,13 +769,19 @@ func (s *jetStreamStorage) PrepareConfigurationStore(memory bool, replicas int)
 
 	kv, err = js.KeyValue(LeaderElectionBucketName)
 	if err == nats.ErrBucketNotFound {
-		kv, err = js.CreateKeyValue(&nats.KeyValueConfig{
+		cfg := &nats.KeyValueConfig{
 			Bucket:      LeaderElectionBucketName,
 			Description: "Choria Async Jobs Leader Elections",
 			Storage:     storage,
 			Replicas:    replicas,
 			TTL:         10 * time.Second,
-		})
+		}
+
+		if maxBytesSet {
```

Comment:
there is a mix of approaches here, in one place you check for `> -1` (line 758) and here you check if its set. I dont think the isSet here really add value

I think now that we never rely on default down here etc we can probably just act on `> -1` as you did above.  Eitherway we should be consistent whichever we pick